migtests/tests/oracle-tests/ora-datatypes-test/env.sh

```diff
@@ -0,0 +1,4 @@
+export SOURCE_DB_TYPE="oracle"
+export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"ORCL"}
+export TARGET_DB_HOST=${TARGET_DB_HOST:-"10.9.203.12"}
+export TARGET_DB_NAME=${TARGET_DB_NAME:-"datatypes_test"}
```

Comment:
I think this might be your personal environment variables - `TARGET_DB_HOST` and `TARGET_DB_NAME`?